tsconfig.base.json

```diff
@@ -16,9 +16,15 @@
     "baseUrl": ".",
     "paths": {
       "@portal-ascepa/angular-plugin": ["plugins/angular/src/index.ts"],
+      "@portal-ascepa/carrousel": [
+        "modules/ultimas-noticias/carrousel/src/index.ts"
+      ],
       "@portal-ascepa/home": ["modules/home/src/index.ts"],
       "@portal-ascepa/layout": ["modules/layout/src/index.ts"],
-      "@portal-ascepa/shared-ui": ["modules/shared-ui/src/index.ts"]
+      "@portal-ascepa/shared-ui": ["modules/shared-ui/src/index.ts"],
+      "@portal-ascepa/ultimas-noticias": [
```

Comment:
Opa, como esse é um componente que vai somente ser utilizado na HomePage, não há a necessidade de criar um module pra ele. 
Você pode usar o componente no módulo `home` que já foi criado em `modules/home/src/lib`

Reply:
This really is it!
No caso, precisaria somente criar o componente em vez de uma lib. Correto?
Por curiosidade, existiria o comando para criar um componente com o Generator do projeto?

Reply:
Fale mestre, entendi. vou fazer as alterações sugeridas. 
Tive exatamente essa duvida na hora de começar e criei o modulo.